R/end_rains.R

```diff
@@ -22,7 +22,7 @@
 #' # is same as R-Instat
 end_rains <- function(data, date_time, station = NULL, year = NULL, rain = NULL,
                       doy = NULL,  s_start_doy = NULL, drop = TRUE,
-                      start_day = 1, end_day = 366, output = c("doy", "date", "both"),
+                      start_day = 1, end_day = 366, output = c("doy", "date", "status"),
```

Comment:
This code used to be you get doy, date, or both. I've edited the functionality here to allow it to be doy, date, and/or status.

Reply:
This is just that the old version of this R function wasn't working as we want it to - you definitely wouldn't be expected to know that!